projects/plugins/jetpack/_inc/client/components/popover/index.jsx

```diff
@@ -76,12 +80,18 @@ class Popover extends Component {
 		bindWindowListeners();
 	}
 
+	isDOMNode( obj ) {
+		return obj instanceof HTMLElement;
+	}
+
 	UNSAFE_componentWillReceiveProps( nextProps ) {
-		// update context (target) reference into a property
-		if ( ! isDOMElement( nextProps.context ) ) {
-			this.domContext = ReactDom.findDOMNode( nextProps.context );
-		} else {
-			this.domContext = nextProps.context;
+		// update context (target - expecting a DOM node not a ref) reference into a property
+
+		if ( this.isDOMNode( nextProps.context ) ) {
+			this.domContextRef.current = nextProps.context;
+		} else if ( nextProps.context !== null ) {
```

Comment:
Could `nextProps.context` be `undefined` (not provided)? It seems like that may be a valid next `context`. Should the error be shown if `context` is not provided?